addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -315,6 +316,80 @@
             </field>
         </record>
 
+        <!-- Inherited form view for mass mailing's form specifically -->
+        <record model="ir.ui.view" id="mailing_mailing_view_full_width_form">
+            <field name="name">mailing.mailing.form_inherit</field>
```

Comment:
Name should globally be the same as xml id (with dots instead of underscores)

Reply:
So `mailing.mailing.view.form.full_width` ?

Reply:
more ``mailing.mailing.view.form.full.width`` (to avoid mix of dots and underscores). As name is not really used on views this is a standard way of filling it.